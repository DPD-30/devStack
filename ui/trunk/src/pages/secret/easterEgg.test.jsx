import React from 'react'
import { renderToString } from 'react-dom/server'

import EasterEgg from './easterEgg.jsx'

jest.unmock('./easterEgg.jsx')


describe('', () => {
    it('EasterEgg should match snapshot', () => {
        const comp = renderToString(<EasterEgg />)

        expect(comp).toMatchSnapshot()
    })
})
