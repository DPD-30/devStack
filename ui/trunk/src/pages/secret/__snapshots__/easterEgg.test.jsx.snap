// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EasterEgg should match snapshot 1`] = `
<img alt="hi"
     class="img-fluid"
     src="easteregg.jpg"
     data-reactroot
>
`;
